setup.py

```diff
@@ -7,15 +7,18 @@
 with open("README.md", "r") as fh:
```

Comment:
Would it be a good idea to move `setup.py` into the python folder to keep all python related code under the python folder? This will require an update to the `packages=setuptools.find_packages()` configuration in `setup.py` to point to the current directory.

Reply:
We will address moving setup.py and requirements.txt in a later ticket.